apps/web/app/layout.tsx

```diff
@@ -101,6 +100,7 @@ export default async function RootLayout({ children }: { children: React.ReactNo
               }
             : {}
         }>
+        <IconSprites />
```

Comment:
having this (svg) in `head` throws an error in Next 15
<img width="961" alt="Screenshot 2024-10-26 at 21 51 19" src="https://github.com/user-attachments/assets/17447ab9-678e-495b-8b58-236f7d06114d">